docs/user_guide/assessment.rst

```diff
@@ -258,6 +258,241 @@ across groups and also the difference and ratio between the maximum and minimum:
     >>> print("ratio in recall = ", grouped_metric.ratio(method='between_groups'))    
     ratio in recall =  0.0
 
+
+Common fairness metrics
+-----------------------
+In the sections below, we review the most common fairness metrics, as well
+as their underlying assumptions and suggestions for use. Each metric requires
+that some aspects of the predictor behavior be comparable across groups. In 
+the mathematical definitions below, let :math:`X` denote a feature vector 
+used for predictions, :math:`A` be a single sensitive feature (such as age 
+or race), and :math:`Y` be the true label. Parity metrics are phrased in 
+terms of expectations with respect to the distribution over :math:`(X,A,Y)`.
+
+.. _demographic_parity:
+
+Demographic parity
+~~~~~~~~~~~~~~~~~~
+Demographic parity is a fairness metric whose goal is to ensure a machine 
+learning model's predictions are independent of membership in a sensitive 
+group. In other words, demographic parity is achieved when the probability 
+of a certain prediction is not dependent on sensitive group membership. In 
+the binary classification scenario, demographic parity refers to equal 
+selection rates across groups. For example, in the context of a resume 
+screening model, equal selection would mean that the proportion of 
+applicants selected for a job interview should be equal across groups.
+
+We mathematically define demographic parity using the following 
+set of equations.
+A classifier :math:`h` satisfies demographic parity under a distribution 
+over :math:`(X, A, Y)` if its prediction :math:`h(X)` is statistically
+independent of the sensitive feature :math:`A`. This is equivalent to
+:math:`\E[h(X) \given A=a] = \E[h(X)] \quad \forall a`. :footcite:`agarwal2018reductions`
+
+In the case of regression, a predictor :math:`f` satisfies demographic parity
+under a distribution over :math:`(X, A, Y)` if :math:`f(X)` is independent
+of the sensitive feature :math:`A`. This is equivalent to
+:math:`\P[f(X) \geq z \given A=a] = \P[f(X) \geq z] \quad \forall a, z`.
+:footcite:`agarwal2019fair` Another way to think of demographic parity in a 
+regression scenario is computing the `mean_prediction()` accross groups 
+and determining how different mean predictions are for different groups. 
+Note that in the Faiurlearn API, :method:`demographic_parity_difference` 
```

Comment:
Could you check whether this will actually refer to the right function, or should we use something like :method:`fairlearn.metrics.demographic_parity_difference` ?

Reply:
Ah, my apologies for the incorrect reference, it's been edited.